docs/tx-malleability.md

```diff
@@ -88,6 +93,8 @@ need to be in the signature bytes themselves.
 Since signatures are not signed over, it is impossible for any sign mode to address this directly
 and instead signature algorithms need to take care to reject any non-canonically encoded signature bytes
 to prevent malleability.
+For the known malleability of the `Multisignature` type, we should make sure that any valid signatures
```

Comment:
was this done by design or just something that fell through the cracks?

Reply:
Pretty sure this just fell through the cracks. We were trying to ensure that transaction contents weren't malleable primarily and we knew that protobuf encoding was non-deterministic. We weren't thinking carefully about how other sources of malleability would affect transaction hashes. In hindsight, I wish we had known all of this, but it is a fair bit to wrap your head around and it took a while to realize all the implications. I imagine there are still things I haven't thought of that aren't in this doc, but I hope we're getting close to having a comprehensive picture.